lib/definitions.py

```diff
@@ -1,16 +1,32 @@
 from dataclasses import dataclass
+# note that it's typing.OrderedDict, not collections.OrderedDict
+from typing import OrderedDict
 import string
 
 
+@dataclass
+class DocRef:
+    type: string
+    name: string
+    # if type is "enum_value"
+    enum_name: string = None
+    enum_value_name: string = None
+    # if type is "class_method"
+    class_name: string = None
+    class_method_name: string = None
+
+
 @dataclass
 class DocItem:
     type: string
     text: string = None
-    values: list[list['DocItem']] = None
+    # presense of this field depends on item type
+    child_items: list[list['DocItem']] = None
 
 
 @dataclass
 class DocComment:
+    # list of lines, each line is list of items
```

Comment:
nit:
maybe `list of paragraphs, each paragraph is list of items`?

or we can introduce new class `DocParagraph`

e.g.

```
@dataclass
class DocParagraph:
    items: list[DocItem]


@dataclass
class DocComment:
    paragraphs: list[DocParagraph]
```